src/node.cc

```diff
@@ -1598,6 +1602,12 @@ void Hrtime(const FunctionCallbackInfo<Value>& args) {
   fields[2] = t % NANOS_PER_SEC;
 }
 
+void HrtimeBigInt(const FunctionCallbackInfo<Value>& args) {
+  Local<ArrayBuffer> ab = args[0].As<BigUint64Array>()->Buffer();
+  uint64_t* fields = static_cast<uint64_t*>(ab->GetContents().Data());
+  fields[0] = uv_hrtime();
```

Comment:
I’d also appreciate some benchmarking showing that the `ArrayBuffer`-based approach is faster than just returning the bigint.

Reply:
@TimothyGu There are no APIs for constructing a BigInt out of a uint64_t at the moment, it seems.

Reply:
I could add one for uint64_t, but there is a discussion going on about the general API design of BigInt: https://bugs.chromium.org/p/v8/issues/detail?id=7712